stories/components/countdown.stories.js

```diff
@@ -8,22 +8,16 @@ storiesOf('Components', module).addWithChapters('Countdown', {
       sections: [
         {
           title: 'Default',
-          sectionFn: () => <Countdown date={Math.round(new Date() / 1000) + 5000} />,
+          sectionFn: () => <Countdown date={5} />,
```

Comment:
`date={5}` seems quite non-explicit. 5 what? seconds? 5 months? I think if we want to change the way that this works then we should rename the prop to something that describes what is expected. `5` is not a `date` which is what I would expect to pass in to a prop named `date`

Reply:
updated